docs/guides/device-gateway/sdk.md

```diff
@@ -0,0 +1,344 @@
+
+![alt-text](img/CNAME-device-gw.png)
+
+Then add another record for the
+[ACME (Automated Certificate Management Environment)](https://venafi.com/blog/what-acme-protocol-and-how-has-it-changed-pki/)
+validation. For the record, enter `_acme-challenge.{SUBDOMAIN}`, and use the value of the `acme_challenge_cname_target` property from from previous response for the value. For the example above, the record name would be
+`_acme-challenge.sitea`.
+
+### Verify DNS
+
+Use the ngrok dashboard to verify that you’ve configured DNS correctly.
+
+1. Login to the [ngrok dashboard](https://dashboard.ngrok.com/).
+2. Click **Domains** in the left-hand navigation menu.
+3. Click on your wildcard domain under _Domain_.
+4. Click **2 targets** next to _DNS Targets_ in the panel displayed on the right-hand side of the screen,
+   as denoted by the arrow in the screenshot below.
+
+![alt-text](img/2tunnels.png)
+
+5. Click **Check Status**, as denoted by the arrow in the screenshot below.
+
+![alt-text](img/check-status.png)
+
+6. You should see a successful response similar to the screenshot below.
+
+![alt-text](img/dns-success.png)
+
+## Create a bot user
+
+First, you’ll create a bot user so that in the next section, you can create an agent authtoken independent of any user account.
+A bot user does not belong to a particular user account. Run the following command to create a new bot user, providing your API key and a description:
+
+```
+curl \
+-X POST \
+-H "Authorization: Bearer {API_KEY}" \
+-H "Content-Type: application/json" \
+-H "Ngrok-Version: 2" \
+-d '{"name":"new bot user from API"}' \
+https://api.ngrok.com/bot_users
+```
+
+You should receive a 201 response similar to the following:
+
+```json
+{
+	"id": "bot_2fmwUXhnImKswMgZKCsx7cnbt2l",
+	"uri": "https://api.ngrok.com/bot_users/bot_2fmwUXhnImKswMgZKCsx7cnbt2l",
+	"name": "new bot user from API",
+	"active": true,
+	"created_at": "2024-04-29T19:39:36Z"
+}
+```
+
+## Create the agent authtoken
+
+You should start each agent using a separate authtoken, and that token should belong to a bot user.
+
+To create an authtoken, login to the ngrok dashboard and click [**Authtokens**](https://dashboard.ngrok.com/tunnels/authtokens)
+under _Tunnels_, then click **Add a Tunnel Authtoken**.
+For **Owner**, select the bot user you use created, and select `bind:*.sitea.{YOUR_DOMAIN}` for the ACL Rules.
+This ACL will allow an agent with the authtoken to create tunnels on any subdomain of `sitea.{YOUR_DOMAIN}`.
+
+![alt-text](img/authtoken.png)
+
+## Configure the sample agent
+
+The sample application uses the ngrok Python SDK to embed connectivity directly into the application
+rather than using the standalone ngrok agent, the CLI, or the ngrok Operator. It serves a local REST
+API you can use to create, delete, and list tunnels.
+
+### Clone the repo
+
+Clone [this sample application](https://github.com/ngrok-samples/ngrok-iot-agent) on the device or device gateway you wish to connect to.
```

Comment:
I believe this should be updated to the new repo you mentioned above?

https://github.com/ngrok-samples/ngrok-python-iot-agent

Reply:
Agreed. And the new repo should have a README. Which, I'm pretty sure we could just use the same README as the original repo. Right, @DevMandy ?